pkg/manila/job.go

```diff
@@ -62,12 +70,13 @@ func DbSyncJob(instance *manilav1.Manila, labels map[string]string, annotations
 		},
 	}
 
-	args := []string{"-c", DBSyncCommand}
+	delayCommand := fmt.Sprintf("sleep %d", delay)
+	args := []string{"-c", fmt.Sprintf("%s && %s", delayCommand, jobCommand)}
```

Comment:
@gouthampacha the resulting command is the concatenation between the delay and the jobCommand.